Bottom: 4c22cb1fe8c648248fc097a062d4ef58f9252a2f
Top:    977b6532f5d7f76626a504f0f22614b31014d460
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 21:27:44 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/fsl/efika-audio-fabric.c b/sound/soc/fsl/efika-audio-fabric.c
index b93e3b9..5afb435 100644
--- a/sound/soc/fsl/efika-audio-fabric.c
+++ b/sound/soc/fsl/efika-audio-fabric.c
@@ -1,5 +1,6 @@
 /*
- * Efika driver for the PSC of the Freescale MPC52xx configured as AC97 interface
+ * Efika driver for the PSC of the Freescale MPC52xx
+ * configured as AC97 interface
  *
  * Copyright 2008 Jon Smirl, Digispeaker
  * Author: Jon Smirl <jonsmirl@gmail.com>
